### PR TITLE
Add boilerplate list

### PR DESCRIPTION
**Pull request by robinheinze**
Issue: https://github.com/infinitered/ignite/issues/943

Adds list of boilerplates (3rd party and IR supported) to the README

### CODE REVIEW

File: BOILERPLATES.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## 3rd Party
+
+| Name | Description | Author |
+|------|-------------|--------|
+| [ignite-native-base-boilerplate](https://github.com/infinitered/ignite-ir-next) | An ignite boilerplate with NativeBase screens | [NativeBase](https://github.com/GeekyAnts) |
+| [ignite-jhipster](https://github.com/ruddell/ignite-jhipster) | A React Native boilerplate for JHipster apps. | [John Ruddell](https://github.com/ruddell) |
+| [ignite-appclon-mobx](https://github.com/Appclon/ignite-appclon-mobx) | Appclon + Ignite Boilerplate | [Appclon](https://github.com/Appclon) |
+
+## Maintained by IR
```
Comment (kevinvangelder) [inline BOILERPLATES.md]: Do we want to list ours first so they don't get lost?

Comment (robinheinze) [inline BOILERPLATES.md]: Yeah, that's not a bad idea. I can switch the order on the PLUGINS page too.

**Comment by GantMan** [comment]
@robinheinze - go ahead and merge master to get this CI to fix.

**Comment by jamonholmgren** [comment]
Thanks @robinheinze !

**Pull request merged**